pkg/resources/tag_association.go

```diff
@@ -116,9 +119,26 @@ func CreateTagAssociation(d *schema.ResourceData, meta interface{}) error {
 		return errors.Wrapf(err, "error associating tag to object: [%v] with command: [%v], tag_id [%v]", objectIdentifier, q, tagID)
 	}
 
-	_, err = snowflake.ListTagAssociations(builder, db)
-	if err != nil {
-		return errors.Wrap(err, "error validating tag association")
+	skipValidate := d.Get("skip_validation").(bool)
```

Comment:
im wondering where this came from. could you please rebase to main?

Reply:
Sure! I think this came from other PR and I failed to operate merging other one. I'll check this carefully then fix around this.

Reply:
@sfc-gh-swinkler Again, thank you for the review. I checked changes around these lines and found just my deleting blank two lines. Does this problem come from some where else...?